### DISCUSSION TITLE
Div breaks continued list

### DISCUSSION THREAD

**Discussion by finkelshtein** [quarto-dev/quarto-cli#10924] [Q&A] [answered]
### Bug description

If a list item contains a div-element followed by a text, the latter text is not indented.

### Steps to reproduce

````qmd

---
title: "test"
format: html
---

* We can see then

:::{.divname}

$$
  a+b=c
$$

:::

    as requested.

* Consider now...

````

### Expected behavior

I'd like to have "as requested" aligned with "We can see". Without div-block it works, even with Latex-formula, i.e. 

```` qmd
* We can see then

    $$
    a+b=c
    $$

    as requested.

* Consider now...
````
works as expected. 

### Actual behavior

The text after div ("as requested") is not indented. Moreover, it is recognised as a quote I believe.

### Quarto check output

Quarto 1.5.57
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.5.57
      Path: C:\Users\dmitri\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: Installation From Path
      Path: C:\texlive\2022\bin\win32
      Version: 2022

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.4.1
      Path: C:/PROGRA~1/R/R-44~1.1
      LibPaths:
        - C:/Users/dmitri/AppData/Local/R/win-library/4.4
        - C:/Program Files/R/R-4.4.1/library
      knitr: 1.48
      rmarkdown: 2.28

[>] Checking Knitr engine render......OK

**Comment by mcanouil** [comment, marked as answer]
There is no bug.
You did not align the sub-elements as documented (see second example in <https://quarto.org/docs/authoring/markdown-basics.html#lists>).

Since you are using empty lines, it means you want a list that is not "compact".
Aligned, means aligned, i.e.:

````qmd
---
title: "test"
format: html
---

* We can see then

  ::: {.divname}

  $$
    a+b=c
  $$

  :::

  as requested.

* Consider now...
````


In the future, please open a GitHub Discussion rather than a GitHub Issue. (I "converted" it to one)

**Reply by finkelshtein** [reply]
Apologies. I tried this, but, accidentally, messed up with indentations in Quarto code and it didn't work either. 
Many thanks!

**Reply by finkelshtein** [reply]
I'm not sure if I should start a new discussion or I can ask here, as this is a related, probably newbie-style, question. I have the code like

`````qmd

:::: {.callout-important}

## Important text

Remember

::: {.highlight}

$$
a+b=c
$$ {#eq-first}

:::

Then

* if [$b>0$]{.highlight key = "inline"}, then $c>a$;

* if [$b<0$]{.highlight key = "inline"}, then $c<a$.

::::

`````

The spans are not recognised, they are typed as plain text. How to correct this?

**Reply by mcanouil** [reply]
It's not fully reproducible.

What's the "key" attribute is supposed to be?
What's is the `highlight` class?

Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by finkelshtein** [reply]
Apologies again, I should stop asking, not my day today. It appeared that `{.highlight key = "inline"}` does not work, whereas `{.highlight key="inline"}` does work (with no spaces around `=` symbol) . (To your question, `.highlight` and `data-key="inline"` were described in css-file.)

**Reply by mcanouil** [reply]
> were described in css-file

Could you give a link? Or something as stating this unfortunately does not make things clearer.

Regarding the spaces around equal signs, that's a Pandoc limitation which Quarto might alleviate in the future.

**Comment by finkelshtein** [comment]
I probably was not clear, after I removed spaces around `=` in
`key="inline"`, the problem was resolved.

**Reply by cscheid** [reply]
Thanks.

This is unfortunately not something Quarto can control; the behavior comes from Pandoc. We recently added a workaround for this problem in divs, but your usecase highlights the same problem in spans.

Even more unfortunately, I think it's harder to work around this problem in spans than it is in divs; I'm not sure we'll be able to do much about it.
